Lib/pdb.py

```diff
@@ -686,6 +686,19 @@ def set_convenience_variable(self, frame, name, value):
     # Generic completion functions.  Individual complete_foo methods can be
     # assigned below to one of these functions.
 
+    def completenames(self, text, line, begidx, endidx):
+        # Overwrite completenames() of cmd so for the command completion,
+        # if no current command matches, check for expressions as well
+        dotext = 'do_' + text
+        commands = [a[3:] for a in self.get_names() if a.startswith(dotext)]
```

Comment:
Use the superclass implementation, don't copy it.
```suggestion
        commands = super().completenames(text, line, begidx, endidx)
```